tools/Makefile.am

```diff
@@ -60,7 +60,7 @@ nut-scanner-deps-usb: $(GENERATED_USB_FILES)
 
 # The distributed nut-snmpinfo.py.in template is assumed to only differ from
 # a generated nut-snmpinfo.py by the @PYTHON@ shebang.
-$(GENERATED_SNMP_FILES): $(top_srcdir)/drivers/*-mib.c
+$(GENERATED_SNMP_FILES): $(top_srcdir)/drivers/*-mib.c $(top_srcdir)/tools/nut-snmpinfo.py.in
```

Comment:
Is that really generated?  It is checked in to git.

Reply:
That's not assignment, that's dependencies to trigger a build of generated stuff. Also above.

Reply:
ENOTENOUGHCOFEE!